shared/packages/worker/src/worker/workers/genericWorker/expectationHandlers/renderHTML.ts

```diff
@@ -699,6 +699,13 @@ class HTMLRenderer {
 			scale !== undefined && `--zoom=${scale}`,
 			`--outputPath=${escapeFilePath(this.outputPath)}`,
 			`--background=${this.exp.endRequirement.version.renderer?.background ?? 'default'}`,
+			`--executableAliases=${
+				this.worker.agentAPI.config.executableAliases
+					? Object.entries<string>(this.worker.agentAPI.config.executableAliases)
+							.map(([k, v]) => `${k}=${v}`)
```

Comment:
This is perhaps a comment for somewhere else, but I thought of it when seeing this

Perhaps an alias argument would have been written as`mytool` being `/some/path/here --mode=headless`, in which case this parsing will break. It could be that we explicitly dont support this, and say to do that via a bash script if it is needed.

Or what if the command needs to include a `;`? (I cant think of why it would though)

Reply:
I did add a hack to support arguments on the form `--myArg="this is a value with spaces"` (see `getProcessArgv()` implementation), that might/should cover most of the use cases, I think?